tests/pytest/core/test_session.py

```diff
@@ -201,15 +163,16 @@ def test_logged_in_True(app_request):
 
 @pytest.mark.django_db
 def test_logout(app_request):
+    LittlepaySession(app_request, access_token="enrollment_token", access_token_expiry="3600")
     OAuthSession(app_request).claims_result = ClaimsResult(verified={"oauth_claim": True})
-    session.update(app_request, logged_in=True, enrollment_token="enrollment_token")
+    session.update(app_request, logged_in=True)
     assert session.logged_in(app_request)
 
     session.logout(app_request)
 
     assert not session.logged_in(app_request)
-    assert not session.enrollment_token(app_request)
```

Comment:
A note here for @machikoyasuda and @lalver1, just because we looked at this briefly at the end of pairing session yesterday: 

I said at the time that this test shouldn't be responsible for asserting state about the Littlepay session. (Same idea with `test_pre_logout()` in `tests/pytest/oauth/test_hooks.py`.)

I feel like my reasoning was mostly influenced by wanting to try and finish out the commit and maybe hurrying through thinking about this. I looked at this closer and realized it's not hard to check that the Littlepay session attributes were updated correctly, so I went ahead and did that, i.e. maintained the assertion using the refactored code.

Reply:
(see https://github.com/cal-itp/benefits/pull/2910/files#diff-fcf5a55d9174de68ae6f8d1b72f014264bfd81d9647acf0d6a38b9aa58bd7adcR174)

Reply:
I think this is fine / good to keep the asserts since they were relatively easy.

Another approach, since there is also good test coverage of `enrollment_littlepay.Session`, would be to mock/patch that class and just assert it was called with `reset=True` here.